### ISSUE TITLE
Is the model trained using Weighted CE Loss? 

### ISSUE THREAD

**Issue by goelarushi** [zhung2/uvtranse#2]
Hi @zhung2 ,

Thanks for your repo. 
When I retrain the model, I am not able to reproduce results in the paper especially for the zero-shot case (Phrase Detection - R@50/R@100 ~ 10.00/13.00). I am using k=1 and cross-entropy loss without weights. Are the results in the paper trained with weighted loss with the 'so_prior.pkl' file? Or any other suggestions for reproducing zero shot results?  
Thanks to you in advance.

**Comment by zhung2** [comment]
Hello,

We treat k as a hyperparameter for zero-shot case, so it would be better if
you try to set k = 5 or k = 10.

Thanks,
Zih-Siou

**Comment by goelarushi** [comment]
That might solve my problem! Thanks a lot for your response @zhung2 .

**Comment by alice-cool** [comment]
> Hi @zhung2 ,
> 
> Thanks for your repo.
> When I retrain the model, I am not able to reproduce results in the paper especially for the zero-shot case (Phrase Detection - R@50/R@100 ~ 10.00/13.00). I am using k=1 and cross-entropy loss without weights. Are the results in the paper trained with weighted loss with the 'so_prior.pkl' file? Or any other suggestions for reproducing zero shot results?
> Thanks to you in advance.

Dear scholar，
I run train.py but it throws errors. Some files the website didn't provide. Could you share some files? Or provide download link?
Thank you
The two model parameter
![image](https://user-images.githubusercontent.com/16870890/126052781-8459c3ad-04f9-4464-bab4-ebb5773b35e1.png)
The file
![image](https://user-images.githubusercontent.com/16870890/126052784-150ac7a6-4713-434c-9daf-e89f7c0055ab.png)
The file
![image](https://user-images.githubusercontent.com/16870890/126052788-d6dfea92-7218-4b39-9901-9f3bf6e00d27.png)
The file
![image](https://user-images.githubusercontent.com/16870890/126052794-bf74f9b6-9076-4dac-b01c-46cf6dc6185a.png)

**Comment by kent252** [comment]
> That might solve my problem! Thanks a lot for your response @zhung2 .

Hello @goelarushi 
I got the same problem as you, could you show me your config or add any file to get the paper result. I appreciate your help. Thank you.
